wavefront_python_sdk/common/connection_handler.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import, division, print_function
+
+from wavefront_python_sdk.common.utils import AtomicCounter
+
+
+class IConnectionHandler:
```

Comment:
Rename to `ConnectionHandler`. Seems Java'ish to preface with `I` since this is technically a class.